contents/docs/integrate/_snippets/install-flutter.mdx

```diff
@@ -6,63 +6,116 @@ Set your PostHog API key and change the automatic event tracking on if you wish
 
 Remember that the application lifecycle events won't have any special context set for you by the time it is initialized. If you are using a self-hosted instance of PostHog you will need to have the public hostname or IP for your instance as well.
 
-#### Android
+To start, add `posthog_flutter` to your `pubspec.yaml`:
 
-```xml file=AndroidManifest.xml
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.posthog.posthog_flutter_example">
+```yaml file=pubspec.yaml
+# rest of your code
+
+dependencies:
+  flutter:
+    sdk: flutter
+  posthog_flutter: ^4.0.1
+
+# rest of your code
+```
+
+Then complete the set up for each platform:
+
+#### Android setup
+
+For Android, add your PostHog configuration to your `AndroidManifest.xml` file located in the `android/app/src/main`:
+
+```xml file=android/app/src/main/AndroidManifest.xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="your.package.name">
     <application>
-        <activity>
-            [...]
-        </activity>
+        <!-- ... other configuration ... -->
         <meta-data android:name="com.posthog.posthog.API_KEY" android:value="<ph_project_api_key>" />
-        <!-- usually 'https://app.posthog.com' or 'https://eu.posthog.com' -->
```

Comment:
I'd recommend keeping this, usually, people don't know what to do/where to get it.